sunpy/net/tests/test_attr.py

```diff
@@ -212,109 +210,93 @@ def test_attr_metamagic(AIA, HMI):
     assert attr.Attr._attr_registry[Instrument].desc == ['This is AIA, it takes data']
     # The _value_registry on the Attr object does not get cleaned.
     # So by adding it again to the same type, in this case Instrument the list is appended.
-    attr.Attr.update_values({Instrument: [('HMI', 'This is HMI, it lives next to AIA')]})
+    attr.Attr.update_values(
+        {GenericClient: {Instrument: [('HMI', 'This is HMI, it lives next to AIA')]}})
     assert attr.Attr._attr_registry[Instrument].name == [AIA.value.lower(), HMI.value.lower()]
     assert attr.Attr._attr_registry[Instrument].name_long == [AIA.value, HMI.value]
-    assert attr.Attr._attr_registry[Instrument].desc == ['This is AIA, it takes data', 'This is HMI, it lives next to AIA']
+    assert attr.Attr._attr_registry[Instrument].desc == [
+        'This is AIA, it takes data', 'This is HMI, it lives next to AIA']
 
     # Tests the print out for the first two inputs only
-    output = textwrap.dedent(
-    "Attribute Name | Full Name | Description                      \n"
-    "---------------+-----------+----------------------------------\n"
-    "aia            | AIA       | This is AIA, it takes data       \n"
-    "hmi            | HMI       | This is HMI, it lives next to AIA")
-
+    output = 'Instrument\n\nDummy Instrument Class.\n\n\nAttribute Name  Client Full Name            Description           \n-------------- ------- --------- ---------------------------------\n           aia Generic       AIA        This is AIA, it takes data\n           hmi Generic       HMI This is HMI, it lives next to AIA'  # NOQA
```

Comment:
Isn't better using the dedent as we had? it looks nicer to read.

Reply:
I mean it was but then it changed and i was too lazy to work out the correct version to dedent.